src/layouts/Footer.js

```diff
@@ -1,14 +1,135 @@
 import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Link from '@material-ui/core/Link';
+import { Paper, withStyles, Grid, TextField, Button, AppBar, Typography, Toolbar, CssBaseline} from '@material-ui/core';
+
+function Copyright() {
+  return (
+    <Typography variant="body2" color="textSecondary">
+      {'Copyright © '}
+      <Link color="inherit" href="https://material-ui.com/">
+        Your Website
+      </Link>{' '}
+      {new Date().getFullYear()}
+      {'.'}
+    </Typography>
+  );
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'auto',
+    minHeight: '100vh',
+  },
+  main: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginTop: theme.spacing(8),
+    marginBottom: theme.spacing(2),
+  },
+  footer: {
+    position: "fixed",
+    bottom: "0",
+    padding: theme.spacing(3, 2),
+    marginTop: 'auto',
+    backgroundColor:
+      theme.palette.type === 'light' ? theme.palette.grey[200] : theme.palette.grey[800],
```

Comment:
Do we have a light/dark mode? Think this is not the recommended way to implement light/dark mode as well, but I don't think we set up the light/dark mode theming yet.

Reply:
Fixed in the latest push